webapp/static/logo.svg

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Just tried to use [svgomg](https://jakearchibald.github.io/svgomg/) and it saved more bytes 👀

```html
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 440.7 440"><style>.st2{fill:#fff}</style><path fill="#003c57" d="M0 0h440v440H0z"/><path fill="#7ed2ee" d="M61.2 255.3h30.4v28.3H61.2z"/><path class="st2" d="M159.3 103.8c-28.5 0-51.7 23.2-51.7 51.7V284H138v-60.5h28.6v-30.4H138v-37.5c0-11.8 9.6-21.4 21.4-21.4 11.5 0 20.8 9.1 21.3 20.4h30.4c-.5-28.1-23.5-50.8-51.8-50.8zM302 126.3h30.4v28.3H302zM302.1 176.1l-.2 107.9h30.4l.1-107.9z"/><path class="st2" d="M284.1 208.4v-32.3h-30.4l-.1 33.8c-.8 11.1-10.1 19.9-21.3 19.9-11.8 0-21.4-9.6-21.4-21.4v-32.3h-30.4v32.3c0 28.5 23.2 51.7 51.7 51.7 6.9 0 13.6-1.3 19.9-4l1.3-.6v23.1h.2v5.8c0 11.8-9.6 21.4-21.4 21.4-11.5 0-20.8-9.1-21.3-20.4h-30.4c.5 28.1 23.5 50.8 51.7 50.8 28.5 0 51.7-23.2 51.7-51.7v-3.8h.1l.1-69.9v-2.4z"/></svg>
```